compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -2583,33 +2558,12 @@ OMR::Z::CodeGenerator::doRegisterAssignment(TR_RegisterKinds kindsToAssign)
       {
       if (self()->getDebug())
          {
-         TR_RegisterKinds rks = (TR_RegisterKinds)(TR_GPR_Mask | TR_FPR_Mask | TR_VRF_Mask);
-         if (self()->supportsHighWordFacility() && !self()->comp()->getOption(TR_DisableHighWordRA))
-            rks = (TR_RegisterKinds) ((int)rks | TR_HPR_Mask);
+         TR_RegisterKinds rks = (TR_RegisterKinds)(TR_GPR_Mask | TR_HPR_Mask | TR_FPR_Mask | TR_VRF_Mask);
 
          self()->getDebug()->startTracingRegisterAssignment("backward", rks);
          }
       }
 
-   // pre-pass to set all internal control flow reg deps to 64bit regs
-   TR::Instruction * currInst = instructionCursor;
-   if (self()->supportsHighWordFacility() && !self()->comp()->getOption(TR_DisableHighWordRA) && TR::Compiler->target.is64Bit())
```

Comment:
I can't figure out why this code was here in the first place. I can't think of why anything can go wrong so I've removed it. My testing shows there is no issue with this. Perhaps there was an age old bug which was fixed and this code was just left in place?